tests/ui/transmutability/enums/should_respect_endianness.stderr

```diff
@@ -2,7 +2,10 @@ error[E0277]: `Src` cannot be safely transmuted into `Unexpected` in the definin
   --> $DIR/should_respect_endianness.rs:36:36
    |
 LL |     assert::is_transmutable::<Src, Unexpected>();
-   |                                    ^^^^^^^^^^ `Src` cannot be safely transmuted into `Unexpected` in the defining scope of `assert::Context`.
+   |                                    ^^^^^^^^^^
+   |                                    |
+   |                                    `Src` cannot be safely transmuted into `Unexpected` in the defining scope of `assert::Context`.
+   |                                    Reason: DstIsBitIncompatible
```

Comment:
I'd suggest:
```
`Src` cannot be safely transmuted into `Unexpected` in the defining scope of `assert::Context` because at least one value of `Src` isn't a bit-valid value of `Unexpected`.
```

Eventually, we'll want to add more information here about why (namely, by giving an example value of the source type that's not a valid instance of the destination type), but we can punt that to future work.